src/renderer/scss/component/_card.scss

```diff
@@ -103,6 +103,7 @@
 .card__title--file {
   font-family: 'metropolis-bold';
   font-size: 28px;
```

Comment:
Since we are making the tiles bigger, I think we can increase the title size too